app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -15,6 +15,9 @@
   <tr id="puppetclass_<%= puppetclass.id %>_params[<%= lookup_key.id %>]"
     class="fields <%= 'overridden' if overridden %>">
     <%= content_tag(:td, puppetclass.name, :rowspan => lookup_keys.size, :class => "ellipsis") if index == 0 %>
+    <td class="ellipsis param_name">
```

Comment:
I think `param_name` class isn't needed here and might actually lead to issues (iirc, it is used by the js to lookup the param name)